### ISSUE TITLE
Is Redux still revelvent

### ISSUE THREAD

**Issue by Nothing-Works** [fChristenson/My-life-as-a-programmer#6]
Hi Fredrik Christenson,

I am not sure if this is an issue or not, if it's not an issue just close it.

I am just wondering Is Redux still relevant. or is it worth to learn it

**Comment by fChristenson** [comment]
Hi m8!

Redux is still the most popular library for React development even with the new context api in recent releases and the reason for that is that it has been added to most company projects already.

Migrating from Redux to the context api will take time and some will not make the migration so knowing Redux is still very useful.

Have a great day m8!

**Comment by Edza** [comment]
I program in Vue. We have vuex. I've heard it's very similar. Definitely using it in the enterprise js.

Our current problem though is an all/nothing approach. Our vue components are a bit too big, for everything to neatly sit in their store (too much extra work), but at the same time we have some cross-dependencies that are not good enough with events/props. In those cases people are still calling things by reference (vue component reference) instead the store.

Basically, what I wanted to say, is that a store (vuex, redux) is useful even if only parts of the state are there. It doesn't have to be completely pure to be make sense.
